### DISCUSSION TITLE
Accessibility Issue: Option to increase font size of the selection window

### DISCUSSION THREAD

**Discussion by chrisgrieser** [espanso/espanso#1275] [Feature requests and Ideas]
**Is your feature request related to a problem? Please describe.**
If you have weaker eyes and/or a display with high resolution (e.g. Mac retina displays), the text in the selection window becomes too small to read.

**Describe the solution you'd like**
An option to change font size.

**Describe alternatives you've considered**
The only alternative I am aware of is to decrease screen resolution system-wide, which well, has all sorts of other drawbacks (content getting squished in pretty much all other apps.)

**Comment by federico-terzi** [comment]
Hey @chrisgrieser,

Thank you for the feedback! We'll mark it as a feature request :)

Cheers :)
